src/DynamoCoreWpf/UI/GuidedTour/GuidesValidationMethods.cs

```diff
@@ -287,7 +290,7 @@ private static void ClosePackageManager(PackageManagerSearchView packageManager)
         /// </summary>
         /// <param name="sender">PackageManagerSearchViewModel</param>
         /// <param name="e">PropertyChanged</param>
-        private static void PackageManagerViewModel_PropertyChanged(object sender, System.ComponentModel.PropertyChangedEventArgs e)
+        private static void PackageManagerViewModel_PropertyChanged(object sender, System.ComponentModel.PropertyChangedEventArgs e, bool checkPackagesListEnableNextStep)
```

Comment:
if `uiAutomationData` can be accessed, why do we need to add such param?

Reply:
You are right, removing the parameter